core/sql/optimizer/RelGrby.h

```diff
@@ -552,7 +560,9 @@ class GroupByAgg : public RelExpr
   // ---------------------------------------------------------------------
   ItemExpr    * groupExprTree_;
   ValueIdSet  groupExpr_;
-
+  
+  ItemExpr  * extraGrpOrderby_;
+  ValueIdSet  extraOrderExpr_;
```

Comment:
You are right Suresh. I will change it into ValueIdList, and avoid the ugly conversion in the OptPhysRelExpr.cpp